src/app-config.c

```diff
@@ -2,7 +2,7 @@
  *      app-config.c
  *
  *      Copyright 2010 PCMan <pcman.tw@gmail.com>
- *      Copyright 2012-2014 Andriy Grytsenko (LStranger) <andrej@rep.kiev.ua>
+ *      Copyright 2012-2021 Andriy Grytsenko (LStranger) <andrej@rep.kiev.ua>
```

Comment:
Nitpick: I think you want `2012-2014, 2021` instead, because `2012-2021` would mean you've also touched the code in 2015, 2016, 2017, 2018, 2019, and 2020.

Reply:
I prefer to use simple range, anyways I did it everywhere before so it's too late for such details now. :D